filebeat/module/haproxy/log/test/default.log-expected.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "@timestamp": "2018-09-20T15:42:59.000Z", 
+        "fileset.module": "haproxy", 
+        "fileset.name": "log", 
+        "haproxy.client_ip": "127.0.0.1", 
```

Comment:
Using local IPs can be a good trick to mitigate #8400 :slightly_smiling_face: but also will avoid the presence of geoip fields, and I think that we want at least to test that they are present :thinking:

Reply:
To clarify this point, we may want to use a public IP here, what would be more similar to a real event. Using a public IP will fill the geoip data (using a local IP won't, and this is why I said that it wouldn't be then affected by #8400).